### DISCUSSION TITLE
Add F2 and F3 calendar

### DISCUSSION THREAD

**Discussion by ghost** [f1multiviewer/issue-tracker#107] [Feature requests]
Since there's a link in F1MV to add the F1 schedule to your personal calendar in the latest release, it might be handy to link to the F2 and F3 calendar as well?

For instance: https://f2fanatic.wordpress.com/interactive-calendar/

**Comment by f1multiviewer** [comment]
The F1TV schedule for each weekend is now shown, but I really would like to stick to either official sources, or first-party (meaning, self-built and self-hosted) options.
